entrypoint.sh

```diff
@@ -5,54 +5,16 @@ manage() {
     sudo -HEu netbox ./manage.py "$@"
 }
 
-setup_environment_variables() {
-    ALLOWED_HOSTS=${ALLOWED_HOSTS:-localhost}
-
-    DB_NAME=${DB_NAME:-netbox}
-    DB_USER=${DB_USER:-netbox}
-    DB_PASS=${DB_PASS:-netbox}
-    DB_HOST=${DB_HOST:-db}
-    DB_PORT=${DB_PORT:-5432}
-
-    LOGIN_REQUIRED=${LOGIN_REQUIRED:-False}
-
-    : "${SECRET_KEY:?SECRET_KEY needs to be set}"
-}
-
 initialize_config() {
     pushd netbox/ 2>&1 > /dev/null
-    cp configuration{.example,}.py
-
-    # Update allowed hosts
-    local allowed_hosts_raw="$ALLOWED_HOSTS"
-    read -ra allowed_hosts_raw <<<"$allowed_hosts_raw"
-
-    local allowed_hosts=""
-
-    for host in "${allowed_hosts_raw[@]}"; do
-        allowed_hosts="$allowed_hosts '$host',"
-    done
-
-    sed -i "/^ALLOWED_HOSTS =/c\\ALLOWED_HOSTS = [$allowed_hosts ]" configuration.py
-
-    # Update DB configuration
-    sed -i "/# PostgreSQL username/c\\    'NAME': '$DB_NAME'," configuration.py
-    sed -i "/# PostgreSQL username/c\\    'USER': '$DB_USER'," configuration.py
-    sed -i "/# PostgreSQL password/c\\    'PASSWORD': '$DB_PASS'," configuration.py
```

Comment:
The official netbox docker-configuration uses `DB_PASSWORD` instead of `DB_PASS`. Can you update the reference in [`docker-compose.example.yml`](https://github.com/pitkley/docker-netbox/blob/aa8241d0551bd7d17df7d08408003b038c19d66f/docker-compose.example.yml#L21) and in the [readme](https://github.com/pitkley/docker-netbox/blob/bf5b7817403127ee464718d0523e840f552c0dc2/README.md#L14)?

Reply:
No problem. I've just committed the change. I will try and add the list with all variables later. If you want you can postpone the pull request until that's finished.